torch/distributions/__init__.py

```diff
@@ -1,11 +1,29 @@
 r"""
 The ``distributions`` package contains parameterizable probability distributions
-and sampling functions.
+and sampling functions. This allows the construction and optimization of dynamic
```

Comment:
nit: I guess I don't think of the graph as being optimized so much as parameters being optimized via the graph. Maybe reword to something like "This allows construction of stochastic computation graphs and stochastic gradient estimators for optimization."?